openc3/lib/openc3/accessors/form_accessor.rb

```diff
@@ -26,16 +26,7 @@ def self.read_item(item, buffer)
       value = nil
       ary.each do |key, ary_value|
         if key == item.key
-          if value
-            if not Array === value
-              value_temp = []
-              value_temp << value
-              value = value_temp
-            end
-            value << ary_value
-          else
-            value = ary_value
-          end
```

Comment:
Why did you remove all this?  HTML Forms can have the same item multiple times and that forms an array.

Reply:
I wasn't sure what this code was doing. `value` is set to nil so line 29 is always false. Didn't realize you could have multiple values with the same name but I guess that is how you handle checkbox groups.

Reply:
Line 29 is not always false if line 33 or 37 has already ran.
We only return an array if the value should be an array (typically the form contains the same name multiple times).